source/options.html

```diff
@@ -15,16 +15,16 @@ <h2>GitHub Enterprise</h2>
 
 	<h2>Disable features</h2>
 	<p>
-		<strong>Notice:</strong> All JavaScript-based features can be disabled here, however the site might not work correctly since CSS changes are still applied. This is mostly untested and feature names might change.
+		List the features to disable, by <a href="https://github.com/sindresorhus/refined-github/tree/master/source/features" target="_blank">filename</a>. Enable logging to show in the console what features are enabled on each page.
 	</p>
-	<p>
-		List the features to disable, by <a href="https://github.com/sindresorhus/refined-github/tree/master/source/features" target="_blank">filename</a>:<br>
-		<textarea name="disabledFeatures" rows="5" placeholder="For example: &#10;mark-unread &#10;hide-own-stars"></textarea>
+	<p style='font-size: 0.9em'>
```

Comment:
Should use double-quotes. And should be moved to the CSS file instead of inline?